### ISSUE TITLE
Request.js is deprecated as of Feb 2020

### ISSUE THREAD

**Issue by ghost** [pwasem/ui5-middleware-http-proxy#7]
As of Feb 11th 2020, request is fully deprecated. No new changes are expected to land. In fact, none have landed for some time.

See here

https://github.com/request/request

**Comment by pwasem** [comment]
@OTTOxMarc 
Yes, [request](https://github.com/request/request) been deprecated for some time now.
As of now I have no plans for replacing it as it is still working.

If you have a suitable replacement though please feel free to contribute and creating a PR.
